CONTRIBUTING.md

```diff
@@ -26,7 +26,7 @@ If you plan to make a larger change or contribution, please discuss this first i
 
 We strive for a healthy balance between subjective perfection and practical considerations, but we are firmly against doing a quick and incomplete job that will require more follow-up work later.
 
-* Use a modern JDK. At least the latest LTS (Java 17 right now) or newer.
+* Use a modern JDK. At least the latest LTS (Java 21 right now) or newer.
```

Comment:
21 or 17? PR comment indicated change to 17

Reply:
You *must* use 17 to build and you *should* use 21 for development.